examples/front-proxy/docker-compose.yaml

```diff
@@ -43,8 +43,6 @@ services:
           - service2
     environment:
       - SERVICE_NAME=2
-    expose:
-      - "8000"
 
```

Comment:
also this - if its not necessary then i would separate out the change

Reply:
Oh that was an accident. That change shouldn't be in there. I'll take it out.